metadata_service/config.py

```diff
@@ -114,3 +116,39 @@ class LocalConfig(LocalGremlinConfig, Config):
         'title': 'Metadata Service',
         'uiversion': 3
     }
+
+
+# The databuilder expects this to be False currently. We are defaulting to true because the testing expects this
+if bool(distutils.util.strtobool(os.environ.get('IGNORE_NEPTUNE_SHARD', 'False'))):
+    shard_set_explicitly('')
+
+
+class NeptuneConfig(LocalGremlinConfig, Config):
```

Comment:
I can move this to documentation if you all wish.

Reply:
we could leave it here to give people who use neptune for reference.